SUMMARY.md

```diff
@@ -1 +0,0 @@
-docs/README.md
```

Comment:
what is this change?

Reply:
I merged master to make sure your merge would be easier, as it said it was out of date.  It looks like SUMMARY.md didn't have a newline at the end, that's what git diff seems to be reporting, but as a result it seems to think that all the content is new.  I didn't touch the file, so not sure why that's happened.

Reply:
hum weird, it looks like it followed the symlink 🤔
Could you fix it? I'll merge once it's done

Reply:
I'm not entirely sure I know how.  Any suggestions?

Reply:
I'd say `git checkout master -- SUMMARY.md` or something similar

Reply:
Done